extensions/interactions/CodeRepl/CodeReplRulesServiceSpec.js

```diff
@@ -148,6 +148,27 @@ describe('Code REPL rules service', function() {
     });
   });
 
+  describe('\'output contains\' rule', function() {
+      var RULE_INPUT = {
+        x: '1'
+      };
+
+      it('should check if output contains some content', function() {
```

Comment:
@seanlip I have corrected the indentation. And regarding the tests, Were you suggesting to add something like this:
  var RULE_INPUT_1 = {
        x: '1 2 3 4'
    }; and
  var RULE_INPUT_2 = {
        x: '1\n2\n3\n4'
    };